docs/user_guide.md

```diff
@@ -1336,6 +1337,34 @@ usage: CodeChecker cmd runs [-h] [--url PRODUCT_URL]
 List the analysis runs available on the server.
 ~~~~~~~~~~~~~~~~~~~~~
 
+### <a name="cmd-run-history"></a> List of run history (`run-history`)
+
+With this command you can list out the specific run events which happened during
+storage processes under multiple run names.
+
+~~~~~~~~~~~~~~~~~~~~~
+usage: CodeChecker cmd run-history [-h] [-n [RUN_NAME [RUN_NAME ...]]]
+                                   [--url PRODUCT_URL]
+                                   [-o {plaintext,html,rows,table,csv,json}]
+                                   [-e EXPORT_DIR] [-c]
```

Comment:
why do we need the `-e EXPORT_DIR` here? Is the html output supported?

Reply:
@gyorb I will fix that in a separate pull request because this problem related to multiple command line commands: `products list`, `runs`, `sum` etc.

Reply:
I have removed the html output from here and I have created a separate pull request (#1515) to fix this problem for other commands.